framework/src/main/java/org/checkerframework/common/basetype/messages.properties

```diff
@@ -98,6 +98,7 @@ contracts.conditional.postcondition.false.methodref.invalid=Subclass postconditi
 lambda.unimplemented=This version of the Checker Framework does not type-check lambda expressions.
 methodref.inference.unimplemented=This version of the Checker Framework does not type-check method references with implicit type arguments.
 
+invalid.polymorphic.qualifier=Cannot use polymorphic qualifier %s %s
```

Comment:
It seems that the second string is always "in a class declaration".  Should it be inline here, or should some of the strings be changed at call sites in the source code?

Reply:
Now theres a call with a different string.